### DISCUSSION TITLE
如何获取skynet的版本号

### DISCUSSION THREAD

**Discussion by cppbeliever** [cloudwu/skynet#1857] [Q&A]
我从网上下载了一个别人写的游戏服务器,用的是skynet，但是我不知道他生成的skynet二进制可执行文件是对应的哪个skynet版本的代码。求解惑。

**Comment by terry8210** [comment]
已经编译成二进制？可以尝试看看使用的lua版本。

**Comment by cppbeliever** [comment]
@terry8210  是的 ，已经编译成了二进制。lua版本是51

**Comment by hanxi** [comment]
看 HISTORY.md

**Comment by cppbeliever** [comment]
@hanxi 没有history.md文件，这是别人的代码，可能被删除了。

**Comment by cppbeliever** [comment]
没有其他办法获取已经编译好的skynet二进制文件的版本号了吗？  执行 ./skynet -v   也不行，没有提高-v这个参数的输出。

**Comment by terry8210** [comment]
@cppbeliever 51，从更新日志看，起码是v1.0.0-alpha以前把

**Comment by cppbeliever** [comment]
还有其他办法获取skynet(已经编译成二进制文件) 的版本号吗？

**Comment by hanxi** [comment]
用 diff 工具对比代码目录，一个个版本对比下就行，基本一年才发一次版本。

**Comment by terry8210** [comment]
我觉得直接1.0的版本来尝试就行了

**Comment by qhf566** [comment]
谁有大厂,近5年内开发完的,国内没有运营过的,成品开发完毕的手机游戏,联系我809996677
